scripts/openstack-quickstart-demosetup

```diff
@@ -787,6 +790,36 @@ if [ "x$with_gnocchi" = "xyes" ]; then
     #start_and_enable_service openstack-gnocchi-statsd
 fi
 
+#-----------------------------------------
+# setup Monasca
+#-----------------------------------------
+
+if [ "x$with_monasca" = "xyes" ]; then
+# monasca-log-agent
+    cp /etc/monasca-log-agent/agent.conf.sample /etc/monasca-log-agent/agent.conf
+    c="/etc/monasca-log-agent/agent.conf"
+    #TODO(sjamgade): change these the following two ips to $IP
+    sed -i "s:\(monasca_log_api_url =>\).*:\1 \"http\://$MONASCA_HOST\:8070/v3.0\":" $c
```

Comment:
The conf is not exactly an ini file. 
Its a custom format, made of blobs of yaml